read-fonts/src/tables/aat.rs

```diff
@@ -0,0 +1,619 @@
+//! Apple Advanced Typography common tables.
+//!
+//! See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6Tables.html>
+
+include!("../../generated/generated_aat.rs");
+
+/// Predefined classes.
+///
+/// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6Tables.html>
+pub mod class {
+    pub const END_OF_TEXT: u8 = 0;
+    pub const OUT_OF_BOUNDS: u8 = 1;
+    pub const DELETED_GLYPH: u8 = 2;
+}
```

Comment:
Would an enum make more sense? Are we going to handle the user class codes?

Reply:
These are predefined classes that are meant to be provided by the shaping engine. Each AAT (sub)table can provide its own glyph classes (through lookup tables) in addition to these so the set is unbounded.